### ISSUE TITLE
WIP: Capsicum (Bell pepper), Winter Cabbage, Crookneck

### ISSUE THREAD

**Issue by andru** [openfarmcc/open-crop-icons#13]
![image](https://cloud.githubusercontent.com/assets/30393/12974348/876da572-d0b3-11e5-9c33-18790436ca95.png)

I'm not too happy with the cabbage or the butternut yet, so any ideas for improvements would be great.

**Comment by jsimplicio** [comment]
Maybe the cabbage needs a bit more leaves?

**Comment by andru** [comment]
Yeah I agree @jsimplicio. I tried it and I couldn't get the leaves to look good, but right now I think it looks more like a deformed brussel sprout. I'll give it another go with outer leaves.

**Comment by simonv3** [comment]
Save it for the brussel sprout though!

**Comment by jsimplicio** [comment]
Maybe greener. Yours has a tint of blue too. I found one online that has a pretty easy to replicate shape and looks good http://thumb7.shutterstock.com/display_pic_with_logo/144271/266208365/stock-vector-vector-paint-hand-drawn-picture-of-cabbage-266208365.jpg

**Comment by andru** [comment]
Sweet, that's a great form. I'll work from that and see what I can do.
